index.js

```diff
@@ -14,7 +14,6 @@ module.exports = {
     'accessor-pairs': 2,
     'array-bracket-spacing': 2,
     'array-callback-return': 2,
-    'arrow-body-style': 2,
```

Comment:
It looks like we're including other allowed rules, so... I guess we want to explicitly allow this.